lib/screens/lobby/lobby.dart

```diff
@@ -34,27 +38,22 @@ class LobbyScreen extends StatelessWidget {
           )
         ],
       ),
-      body: Container(
-        child: StreamBuilder<List<Game>>(
-            stream: context.watch<DatabaseService>().streamOfCreatedGames,
-            builder: (context, snapshot) {
-              if (snapshot.data == null) {
-                return Container();
-              } else {
-                return Padding(
-                  padding: const EdgeInsets.fromLTRB(0, 10, 0, 0),
-                  child: ListView.builder(
-                    itemCount: snapshot.data.length,
-                    itemBuilder: (context, index) {
-                      return GameTile(snapshot.data[index], index);
-                    },
-                  ),
-                );
-              }
-            }),
-      ),
+      body: FutureBuilder<Position>(
+          future: context.watch<LocationService>().getUsersCurrentLocation,
+          builder: (context, snapshot) {
+            if (snapshot.hasData == false) {
+              return Loading();
+            } else {
+              final userLocation = Location(
```

Comment:
location will be passed to list of created games to do distance calculations in #60

Reply:
sweet!